tests/test_util.py

```diff
@@ -11,17 +13,29 @@
 from josepy import ComparableRSAKey, ComparableX509
 from josepy.util import ComparableECKey
 
+# This approach is based on the recommendation at
+# https://github.com/python/mypy/issues/1153#issuecomment-1207333806.
+if sys.version_info >= (3, 9):
+    import importlib.resources as importlib_resources
+else:
+    import importlib_resources
+
+TESTDATA = importlib_resources.files('testdata')
+
 
 def vector_path(*names: str) -> str:
     """Path to a test vector."""
-    return pkg_resources.resource_filename(
-        __name__, os.path.join('testdata', *names))
+    file_manager = contextlib.ExitStack()
```

Comment:
It would be nice to have an explanatory comment here.

For me, it was not obvious that `as_file` might generate temporary files on disk that need to be cleaned up.

Reply:
That's totally reasonable. I did something slightly different but maybe more useful; I linked to the migration guide I based this code on.